taegis-xdr/info.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "taegis-xdr",
   "label": "Taegis XDR",
-  "version": "1.0.0",
-  "description": "SecureWorks Taegis™ XDR offers superior detection, unmatched response and an open platform built from the ground up to integrate market-leading technologies and deliver the highest ROI.",
+  "version": "1.1.0",
```

Comment:
Arrange the actions as per the crud sequence

Reply:
Resolved